tools/windows-node-installer/test/e2e/azure_test.go

```diff
@@ -215,6 +224,19 @@ func areRequiredRulesPresent(secGroupRules []network.SecurityRule) bool {
 	return true
 }
 
+// getIpPass extracts instance IPAddress and password from instance rdp file.
+func getIpPass(vmName string) (string, string, error) {
```

Comment:
This function doesnt have enough documentation. There needs to be a comment describing the format of the instance rdp file.